bin/sdk/gutenberg.js

```diff
@@ -2,26 +2,28 @@
 /**
  * External dependencies
  */
+const fs = require( 'fs' );
 const path = require( 'path' );
 
-exports.config = ( {
-	argv: { editorScript, viewScript, outputDir, outputEditorFile, outputViewFile },
-	getBaseConfig,
-} ) => {
+const EDITOR_FILENAME = 'editor';
+const VIEW_FILENAME = 'view';
```

Comment:
are these giving us much utility? at thirty lines in the module I'm thinking some of these things would even be easier without the constants.

```js
const editScript = path.join( inputDir, 'edit.js' );
const viewScript = path.join( inputDir, 'view.js' );

…

...( fs.existsSync( editScript ) ? { edit: editScript } : {} ),
...( fs.existsSync( viewScript ) ? { view: viewScript } : {} ),
```